build/custom-ci-Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openshift/origin-release:golang-1.14
+RUN yum -y install epel-release --disablerepo=epel && yum clean all
+RUN yum -y install python3-pip && yum clean all
+RUN pip3 install -U pip
+RUN pip3 install ansible jmespath molecule more-itertools openshift yamllint
```

Comment:
Are we using molecule tests for oadp-operator?

Reply:
Yes, based on what I discussed with Dylan. But I am always open to suggestions.

Reply:
@shawn-hurley I asked Hiteshwari to integrate openshift CI with calling out to molecule so that we could leverage it like mig-operator if desired.

It is likely that we change this to simply be an e2e test suite written in golang... so I will hold off on merging until we've made that decision.